core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -228,12 +228,17 @@ class KafkaApis(val requestChannel: RequestChannel,
       val invalidRequestsInfo = offsetCommitRequest.offsetData.asScala.filter { case (topicPartition, _) =>
```

Comment:
We can use `partition` here too to simplify the code. And the variable should probably be named `nonExistingRequestsInfo`.